user-guide/Troubleshooting/Known_issues/KI_DataMinerMessageBroker.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: KI_DataMinerMessageBroker
+---
+
+# Max Payload exceptions occur when using MessageBroker with chunking
+
+## Affected versions
+
+From DataMiner 10.3.5 [CU0]/10.3.8 onwards.
+
+## Cause
+
+An issue can occur when MessageBroker uses chunking, which employs the memory allocation method [Array Pooling](https://learn.microsoft.com/en-us/dotnet/api/system.buffers.arraypool-1?view=net-7.0). This method generates 'chunks' to enhance memory usage efficiency. However, in some cases, these generated chunks are not trimmed to the correct size before transmission. This issue is most likely to occur with messages that need to be split into multiple chunks.
```

Comment:
_This method generates 'chunks' to enhance memory usage efficiency._
This sentence doesn't make that much sense in this context and isn't that important. Can be removed imo.

Reply:
Done in commit [e705760](https://github.com/SkylineCommunications/dataminer-docs/pull/1995/commits/e705760295b3bb2b7ab0b049adc593b99a1898f7)